testhelper/testhelper.go

```diff
@@ -48,42 +47,9 @@ func ResetFolder(path string) error {
 	return os.Mkdir(path, dirInfo.Mode())
 }
 
-// CheckDeleteProcessedData compares the number of files found in a specified data
-// directory with the previous number found and uses the useLiveData and
-// deleteProcessedData values to evaluate this comparison. It returns the number of files
-// currently in the data directory for the specified config. Future invocations should pass in this
-// value. This function should be passed 0 as a default prev argument in order to get the
-// number of files currently in the directory.
-func CheckDeleteProcessedData(t *testing.T, slamMode slam.Mode, dir string, prev int, deleteProcessedData, useLiveData bool) int {
```

Comment:
Why do we remove this function? (Was it just dead code?)

Reply:
We removed this function because it depended on `slam.Mode`, which is deprecated. We decided to call this function in the respective slam repos instead and remove it from here.

Reply:
got it, thanks!